### ISSUE TITLE
Statements inside of html blocks cause crash

### ISSUE THREAD

**Issue by aDogCalledSpot** [schvv31n/yew-fmt#17]
```rust
html! {
    <div class={
        let x = "foo";
        x
    } />
}
```

Leads to:

```
error: unexpected token
  ┌─ /path/to/src/main.rs:8:26
  │
8 │             let x = "foo";
  │                          ^
```

**Comment by schvv31n** [comment]
This is because brackets in `html!` don't define a code block in a way Rust defines it, instead they accept 1 expression, like so
```rs
html! {
    <div class={{
        let x = "foo";
        x
    }} />
}
```

**Comment by aDogCalledSpot** [comment]
The double braces aren't necessary though. The code above runs fine. Here's the full main.rs:

```rust
use yew::prelude::*;

#[function_component(Application)]
fn app() -> Html {
    html! {
        <div class={
            let x = "foo";
            x
        } />
    }
}

fn main() {
    yew::Renderer::<Application>::new().render();
}
```

----

Formatting with double braces works but removes some new lines too many:

```rust
html! { <div class={{
    let x = "foo";
    x
}} /> }
```

**Comment by schvv31n** [comment]
Thank you for pointing this out, this aspect of Yew HTML's syntax was vaguely documented so before looking into the source code of the HTML macros, I didn't know the syntax for block nodes and prop values was different, but it should be fixed now in v0.5.1
